### PR TITLE
doc: gatsby image configuration

### LINKED ISSUE #550: Doc Image Configuration for Site Builder
## User Story

As a site builder, I want a Image Configurations site build guide to use so I can better understand the usage, guidelines & best practices of the image configurations component so that I can quickly start using it.

### Acceptance Criteria

AC

In the Bodiless Documentation, under "Development" and in Site Build, there is a section for "Image Configurations" 

AC

The Image Configurations documentation, include the usage, guidelines & best practices for working with this component.

AC

The documentation clearly states that if the Developer makes any configuration changes to the Image they should manually rebuild. 

### Implementation Notes

Provide list of image presets and breakpoints 

Notes from Andrei
```
1. breakpoints customization.
do we want to provide ability for site builders to customize a list of breakpoints for which images should be generated? currently, the list of breakpoints is hardcoded in bodiless gatsby theme.
I believe long term, we should have a single source of truth for configuring breakpoints for a site. This configuration should be injected to tailwind and bodiless gatsby theme.
```

### References

https://github.com/johnsonandjohnson/Bodiless-JS/issues/477
https://github.com/johnsonandjohnson/Bodiless-JS/pull/531

### PR DESCRIPTION
**Pull request by beliayeu**
## Changes
* added doc how to activate gatsby image on a site

## Related Issues

Closes #550

### CODE REVIEW

File: packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md [hunk 1]
```diff
@@ -706,3 +706,98 @@ const galleryDesign = varyDesign(
...
+ <FluidGatsbyImage />
+```
+
+1. Upload images via BodilessJS admin interface.
+
+> Note: when you activate gatsby image for an existing image, then you need to reupload the image or update the corresponding image json file.
+
+### Change image preset
+
+Lest consider, you have an image with `GatsbyImagePresets.Fluid` preset and you want to change the preset to `GatsbyImagePresets.FluidNoBase64`.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md:772]:
```suggestion
If you have an image with `GatsbyImagePresets.Fluid` preset and you want to change the preset to `GatsbyImagePresets.FluidNoBase64`.
```

File: packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md [hunk 2]
```diff
@@ -706,3 +706,98 @@ const galleryDesign = varyDesign(
...
+
+const asGatsbyImg = (preset: string) => flowRight(
+  withGatsbyImageNode(preset),
+  asBodilessImage(),
+  withGatsbyImageLogger(preset),
+  asBaseGatsbyImage,
+);
+```
+
+> Note: We are using `asGatsbyImage`, `withGatsbyImageNode`, `withGatsbyImageLogger` functions provided by `@bodiless/gatsby-theme-bodiless`.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md:734]:
```suggestion
> Note: BodilessJS uses `asGatsbyImage`, `withGatsbyImageNode`, `withGatsbyImageLogger` functions provided by `@bodiless/gatsby-theme-bodiless`.
```

File: packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md [hunk 3]
```diff
@@ -706,3 +706,98 @@ const galleryDesign = varyDesign(
...
+
+```ts
+import { GatsbyImagePresets } from '@bodiless/gatsby-theme-bodiless';
+
+const asFluidGatsbyImage = asGatsbyImg(GatsbyImagePresets.Fluid);
+const asFluidWithWebpGatsbyImage = asGatsbyImg(GatsbyImagePresets.FluidWithWebp);
+);
+```
+
+> Note: We are using default image presets exposed by `@bodiless/gatsby-theme-bodiless`. Visit [default image presets](#default-image-presets) section to get a list of available presets.
```
Comment (malachye) [inline packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md:749]:
```suggestion
> Note: BodilessJS uses default image presets exposed by `@bodiless/gatsby-theme-bodiless`. Visit [default image presets](#default-image-presets) section to get a list of available presets.
```

File: packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md [hunk 4]
```diff
@@ -706,3 +706,98 @@ const galleryDesign = varyDesign(
 )();
 
 ```
```
[the remark this answers is not in the document]
Comment (malachye) [reply packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md:708]: @hvanyo i agree that 'Building Sites' might be a better title. Or maybe 'Building a Basic Site'? Will ask Kevin and Sophia.

[the remark this answers is not in the document]
Comment (hvanyo) [reply packages/bodiless-documentation/doc/Development/Guides/SiteBuildBasics.md:708]: @malachye  @beliayeu  just to confirm -- this discussion should go into different ticket and is not relevant to this PR.

**Review by malachye — APPROVED** [review]

**Review by hvanyo — APPROVED** [review]

**Review by malachye — APPROVED** [review]

**Pull request merged**